dashboard/db/migrate/20201207190534_change_resource_course_version_id_to_non_nullable.rb

```diff
@@ -0,0 +1,12 @@
+class ChangeResourceCourseVersionIdToNonNullable < ActiveRecord::Migration[5.2]
+  def up
+    if [:development, :adhoc].include? rack_env
+      Resource.where(course_version_id: nil).destroy_all
```

Comment:
I tried this locally on my test db and got an exception about data being truncated. I think this means we need to include `:test` in the list to get the behavior we want.

Reply:
(since i need to run our specific tests, I did `RAILS_ENV=test bundle exec rake db:reset db:test:prepare` and the problem went away).

Reply:
Ah I was wondering about that but I thought I tested it. I'll add it